src/services/deployment-service/deployment-service.ts

```diff
@@ -0,0 +1,87 @@
+import Logger from "bunyan";
```

Comment:
should it be `src/services/deployment-cache-service/deployment-cache-service.ts` maybe?

Reply:
It could be too narrow, what if we want to put more function in this file? I am not too against the idea, but no too fancy about it.

Reply:
Okay, bring service one level up and renamed to `deployment-cache-service.ts`